distribution/ddf-common/src/main/resources/security/default.policy

```diff
@@ -571,4 +571,12 @@ grant {
     permission java.io.FilePermission "/etc/hosts", "read";
     permission java.io.FilePermission "/etc/hostname", "read";
     permission java.io.FilePermission "/etc/resolv.conf", "read";
+
+    //Internationalization resource bundles
```

Comment:
❗️ we shouldn’t be granting everybody permissions to read i18n

Reply:
For clarity it wasn't an indication that we should but that we could add it to the common grant section.

Reply:
Changes to the security policy have been made.